### PR TITLE
Add test ids

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

Adds ids to tests so that they can be found.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/windows_test.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ var _ = Describe("Windows VirtualMachineInstance", func() {
 			}
 		})
 
-		It("should have correct UUID", func() {
+		It("[test_id:240][ref_id:295] should have correct UUID", func() {
```
Comment (ILpinto) [inline tests/windows_test.go]: the ref_if 
should be one time in the 
Context("with kubectl command"

File: tests/windows_test.go [hunk 2]
```diff
@@ -291,7 +291,7 @@ var _ = Describe("Windows VirtualMachineInstance", func() {
 			tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 360)
 		})
 
-		It("should succeed to stop a vmi", func() {
+		It("[test_id:239][ref_id:222] should succeed to stop a vmi", func() {
```
Comment (ILpinto) [inline tests/windows_test.go]: same

**Comment by dhiller** [comment]
@ILpinto changes incorporated Please review again, thanks!

**Comment by ILpinto** [comment]
LGTM

**Pull request merged**